winui/Rating/Precision.md

```diff
@@ -34,7 +34,7 @@ rating.ItemsCount = 5;
 {% endhighlight %}
 {% endtabs %}
 
-![Rating control](Rating_images/winui_rating_precision_full.png)
+![Rating control using whole precision in WinUI](Rating_images/winui_rating_precision_full.png)
```

Comment:
@Priyanka-Vijayasankar WinUI Rating control with full precision. Change like this other precision too.

Reply:
Changed the content as mentioned above and also changed other precision.